### DISCUSSION TITLE
handle clipboard prompt.

### DISCUSSION THREAD

**Discussion by jenishgandhi7** [cypress-io/cypress#21085] [Component Testing]
Any one have idea how to handle this clipboard prompt. i tried multiple ways but able to fix that 
![image](https://user-images.githubusercontent.com/8191015/163223889-b61aa1b4-8aab-43f9-8ab4-a0dd3b907656.png)

**Comment by jjhelguero** [comment]
This looks like a window alert. You can [stub the window alert](https://github.com/cypress-io/cypress/issues/5316#issuecomment-541173970) to prevent the alert from appearing during your test. If I understand correctly this should work.

```js
it('window alert test', function () {
  cy.visit('/index.html', {
    onBeforeLoad(win) {
      cy.stub(win, 'alert').as('windowAlert')
    }
  })
  
  cy.get('#prompt').click().then(function () {
    // you'll add your alert text here
    expect(this.windowAlert).to.be.calledWith('Copy to clipboard: Ctrl+C, Enter')
  })
})
```
